app/main.py

```diff
@@ -1,3 +1,41 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, second: object | int) -> object:
```

Comment:
do not use object as a type annotation. Every type is object in python
Make clear type annotations here: Disctance, integer, float